ethdb/database.go

```diff
@@ -81,6 +81,12 @@ type AncientReader interface {
 
 	// AncientSize returns the ancient size of the specified category.
 	AncientSize(kind string) (uint64, error)
+
+	// ItemAmountInAncient returns the actual length of current ancientDB.
+	ItemAmountInAncient() (uint64, error)
+
+	// AncientOffSet returns the offset of current ancientDB.
+	AncientOffSet() (uint64, error)
```

Comment:
this function never returns an error.

Reply:
Yes, but actually I found that there're many other places used the same way as well, like `Ancients()`, will be a problem?

Reply:
Done